### ISSUE TITLE
Review UX of azd env refresh

### ISSUE THREAD

**Issue by savannahostrowski** [Azure/azure-dev#2751]
Reviewing our in product survey and based on verbatims I've heard from various folks, it's not intuitive that if the developer doesn't already have a local env that they first need to create one as part of the azd env refresh flow, making the prompt we show jarring.

On solution here might be to automatically create one (implicitly) when there are no current local environments for the project, avoiding the prompt.

**Comment by savannahostrowski** [comment]
@Austinauth @gkulin I don't think this is being tackled for the November release. I will reassign for improvements to be released in January?

**Comment by Austinauth** [comment]
I'll add this to my list of items to prioritize for work in Jan - March timeframe. If the team wants to tackle in the interim I can do my best to review and provide feedback.
